c/src/lib/libbsp/arm/shared/stm32fxxxx/include/hal-error.h

```diff
@@ -0,0 +1,28 @@
+/**
+ * @file hal-error.h
+ *
+ * @ingroup error
+ *
+ * @brief Defines generic error handler referenced throughout the STM32F
+ *  HAL code.
+ *
+ */
+
+/*
+ * Copyright (c) 2015 Vecna Technologies, Inc.
+ *
+ * The license and distribution terms for this file may be
+ * found in the file LICENSE in this distribution or at
+ * http://www.rtems.com/license/LICENSE.
+ */
+
+#ifndef INCLUDE_HAL_ERROR_H_
+#define INCLUDE_HAL_ERROR_H_
+
+/**
+ * @brief HAL ST32F error handler function.
+ */
+void Error_Handler(void);
```

Comment:
FYI, this was just a placeholder function that we could start calling in the code whenever there is a "I should never be here"  condition.  I think it should be renamed to something like stm32_error_handler though.